app/serializers/curator/filestreams/audio_serializer.rb

```diff
@@ -2,5 +2,15 @@
 
 module Curator
   class Filestreams::AudioSerializer < Filestreams::FileSetSerializer
+    schema_as_json do
+      node :audio_master_data, target: -> (record) { record.audio_master_blob } do
+        attribute(:id) { |record| record.key }
```

Comment:
is there any way to avoid repeating this block in every serialzer?
```ruby
attribute(:id) { |record| record.key }
node :metadata do
  attributes :byte_size, :checksum
  attribute(:file_name) { |record| record.filename.to_s }
  attribute(:mime_type) { |record| record.content_type.to_s }
end
```

Reply:
Not at the moment. I need to do issue #95 to make that work